### PR TITLE
[Backport stable/1.2] Let `WorkloadGenerator` wait longer for created incident

### PR DESCRIPTION
**Pull request by lenaschoenburg**
Manual backport of #8650

### CODE REVIEW

**Review by npepinpe — APPROVED** [review]
bors merge

File: test/pom.xml [hunk 1]
```diff
@@ -112,6 +112,12 @@
       <artifactId>agrona</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <scope>compile</scope>
```
[the remark this answers is not in the document]
Comment (oleschoenburg) [reply test/pom.xml:118]: It's specified as "test" in our dependency management, hence the overwrite to default.

**Pull request merged**